executor/write.go

```diff
@@ -604,7 +604,7 @@ type InsertExec struct {
 
 	OnDuplicate []*expression.Assignment
 
-	Priority int
+	Priority ast.PriorityEnum
```

Comment:
I add select statement in distsql first, priority for insert will be added later.
The two PRs would conflict on PriorityEnum definition, I'll rebase anther if one of them is merged.